backend/entityservice/cache/progress.py

```diff
@@ -1,13 +1,15 @@
 import structlog
 
-from entityservice.settings import Config as config
+from entityservice.settings import Config as globalconfig
 from entityservice.cache.connection import connect_to_redis
 from entityservice.cache.helpers import _get_run_hash_key
 
 logger = structlog.get_logger()
 
 
-def save_current_progress(comparisons, run_id):
+def save_current_progress(comparisons, run_id, config=None):
```

Comment:
I find it confusing to be able to pass `config` in here, as almost all of it gets ignored anyway. It'd be clearer if it's just a cache expiry value. But then, why would you want to set a different expiry value in the first place?

Reply:
Yeah you'd only want to do that for testing. My logic is that it would be nice to slowly move away from using the global `Config` as we add tests.